docs/docs/erc20_onboarding.md

```diff
@@ -5,95 +5,158 @@ sidebar_label: ERC20 opt-in onboarding
+truffle(develop)> MyLegacyToken.new('MyToken', 'MTK', 18, { from: owner }).then(i => legacyToken = i)
 truffle(develop)> legacyToken.address
 '0x...'
 ```
 
 Keep track of the `owner` and `legacyToken` addresses, we will need them in the following steps.
 
-You can check the old token balance by running:
+You can check the owner balance by running:
 ```sh
 truffle(develop)> legacyToken.balanceOf(owner)
 BigNumber { s: 1, e: 0, c: [ 100 ] }
 ```
 
 Remember not to close this console, as we will be using it later.
 
-### 2. Initialize your ZeppelinOS project
+### 1. Initialize your migration project with ZeppelinOS 
 
 If you haven't installed the ZeppelinOS CLI, run the following command in your terminal:
 ```sh
 npm install --global zos
 ```
 
-To initialize a ZeppelinOS project, open a terminal and run the following lines:
+To initialize this project with ZeppelinOS, open a terminal and run the following line:
+
+```sh
+zos init my-token-migration 1.0.0
+```
+
+We have just initialized a new ZeppelinOS project. You should see a new `zos.json` file with the following content:
+
+```json
+{
+  "name": "my-token-migration",
+  "version": "1.0.0",
+  "contracts": {}
+}
+``` 
+
+Next, we will have to modify the legacy token contract to get the new upgradeable version of it where the current balances
+are going to be migrated. 
+
+Following our sample project, you will find another contract called [`MyUpgradeableToken`](https://github.com/zeppelinos/erc20-opt-in-onboarding/blob/master/contracts/MyUpgradeableToken.sol) 
+which is the upgradeable version of the sample legacy token contract [`MyLegacyToken`](https://github.com/zeppelinos/erc20-opt-in-onboarding/blob/master/contracts/MyLegacyToken.sol):
+
+```solidity
+import "./openzeppelin-zos/MigratableERC20.sol";
+import "openzeppelin-zos/contracts/token/ERC20/DetailedERC20.sol";
+import "openzeppelin-zos/contracts/token/ERC20/StandardToken.sol";
+
+contract MyUpgradeableToken is MigratableERC20, StandardToken, DetailedERC20 {
+
+  function initialize(ERC20 _legacyToken, string _name, string _symbol, uint8 _decimals)
+  isInitializer("MyUpgradeableToken", "1.0.0")
+  public
+  {
+    MigratableERC20.initialize(_legacyToken);
+    DetailedERC20.initialize(_name, _symbol, _decimals);
+  }
+  
+  function _mint(address _to, uint256 _amount) internal {
+    require(_to != address(0));
+    totalSupply_ = totalSupply_.add(_amount);
+    balances[_to] = balances[_to].add(_amount);
+    emit Transfer(address(0), _to, _amount);
+  }
+}
+```
+
+On one hand, we are inheriting from the `MigratableERC20` contract provided by the OpenZeppelin stdlib which provides 
+all the migration functionality we will need. The `MigratableERC20` contract requires the implementation to define an
+internal minting function. In this case we are defining one following the `StandardToken` contract. However, this 
+function will be provided built-in in the upcoming version of OpenZeppelin.     
```

Comment:
What if we include the version of StandardToken that has the mint function in this project, as we are doing with the MigratableERC20? This part is somewhat confusing, and could be removed if we were using the latest version.

Reply:
The thing is that we will have to use another name for it and I'm not sure if it could be more confusing than what be have now :/